staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -685,17 +696,19 @@ const (
 	VirtualMachineInstanceReplicaSetReplicaPaused VirtualMachineInstanceReplicaSetConditionType = "ReplicaPaused"
 )
 
-// ---
+//
 // +k8s:openapi-gen=true
 type VirtualMachineInstanceTemplateSpec struct {
+	// +kubebuilder:pruning:PreserveUnknownFields
```

Comment:
why was this added?

Reply:
Nice catch. That was me testing which markers are detected by kube-openapi. This has no effect as far as I know. Should I remove it? And thanks for go.mod. I have a habit to not commit it.

Reply:
alrighty, i removed it.